### DISCUSSION TITLE
KeyVal default value

### DISCUSSION THREAD

**Discussion by invaders-xx** [filamentphp/filament#2984] [Help]
I am trying to set default KeyVal values within a Repeater with an array of values ['fr'=>'', 'en'=>''] but it always appears empty

**Comment by invaders-xx** [comment]
In fact, my Repeater has ->lazy(). If I remove this parameter it works. Normal behavior ?

**Reply by danharrin** [reply]
Adding `lazy()` to the Repeater will make all items within it `lazy()` as well, and Alpine fields don't support `lazy()` yet (because of a Livewire limitation). So this is probably breaking the KeyVal.

Why do you need `lazy()`?

**Reply by invaders-xx** [reply]
Because I am building a second repeater based on the content of the first repeater

**Reply by danharrin** [reply]
Instead of `->lazy()` on the repeater, you should probably add `->lazy()` to each native HTML input, and then `->reactive()` to the rest
